TDDByExample/money/src/Money.js

```diff
@@ -0,0 +1,52 @@
+import { Expression, Sum } from './Expression.js';
+
+class Money extends Expression {
+  #amount
+  #currency
+  constructor(amount, currency) {
+    super();
+    this.#amount = amount;
+    this.#currency = currency;
+  }
+  get amount() {
+    return this.#amount
+  }
+  times(multiplier) {
+    return new Money(this.amount * multiplier, this.currency);
+  }
+  get currency() {
+    return this.#currency;
+  }
+  equals(another) {
+    if (!(another instanceof Money)) {
+      return false;
+    }
+    if (this.currency !== another.currency) {
+      return false;
+    }
+    return this.amount === another.amount; // 1. private이기 때문에 에러가 발생한다.
+  }
+  toString() {
+    return this.amount + " " + this.currency;
```

Comment:
어째서 템플릿 리터럴을 사용하지 않은 걸까요? ㅎㅎ

간단한 함수라도 암묵적 형변환은 좋지 않다고 생각합니다.
템플릿 리터럴을 사용해주세요~

```suggestion
    return `${this.amount} ${this.currency}`;
```